wp-content/themes/wp-starter/src/styles/blocks/cta.css

```diff
@@ -1,5 +1,16 @@
 @layer components {
 	.acf-block-cta {
+		/* Variants for background color */
+		&.has-black-background-color,
+		&.has-gray-500-background-color,
+		&.has-accent-background-color * {
+			@apply !text-white;
+
+			.wp-block-heading {
+				@apply !text-white;
+			}
+		}
```

Comment:
I think this might be some merge/rebase regression? I thought we found a solution that didn't require `!important`